metadata/indices/h3_hexes_level_5.md

```diff
@@ -0,0 +1,69 @@
+# Title
+
+Utah H3 Hexes Level 5
+
+## ID
+
+4e8fb736-5e1a-4c84-8683-88b0fcc15768
+
+## Brief Summary
+
+## Summary
+
+The [H3 indexing system](https://h3geo.org/docs) provides a standardized and high-performance grid for aggregating data at various levels. This layer contains the spatial representation of the hex IDs that cover the state of Utah for resolution level 5.
```

Comment:
```suggestion
The [H3 indexing system](https://h3geo.org/docs) provides a standardized and high-performance grid for aggregating point data at various levels. This layer contains the spatial representation of the hex IDs that cover the state of Utah for resolution level 5.
```

Reply:
are the h3's limited to aggregating point data?

Reply:
I suppose they can aggregate anything that can be reduced to a point. The H3 API is based on converting a lat/long to an H3 id, so that's why I made the distinction here.